integration-tests/src/test/scala/tech/beshu/ror/integration/suites/MiscSuite.scala

```diff
@@ -81,6 +81,24 @@ trait MiscSuite
     result.searchHits.size should be(1)
     result.searchHits(0)("_source")("user_id").str should be("alice")
   }
+
+  "Cluster health response using response_fields rule should be filtered" in {
+    val dev1ClusterStateManager = new ClusterManager(esTargets.head.basicAuthClient("dev1", "test"), esVersion = esTargets.head.esVersion)
+    val healthCheck = dev1ClusterStateManager.health()
+
+    println(healthCheck.responseJson.toString())
+    healthCheck.responseJson.obj.isDefinedAt("cluster_name") should equal(true)
+    healthCheck.responseJson.obj.isDefinedAt("number_of_nodes") should equal(false)
+  }
+
+  "Cat health response using response_fields rule should be filtered" in {
```

Comment:
we should also write test for:
* nested fields (eg. `user.name` for JSON like this:

```
{
  "user": {
    "name": "...",
    "age": 80
  }
}
```

* many fields